corehq/messaging/scheduling/static/scheduling/js/dashboard.js

```diff
@@ -106,25 +106,17 @@ hqDefine("scheduling/js/dashboard",[
     var dashboardViewModel = new DashboardViewModel();
     dashboardViewModel.init();
 
-    var updateDashboard = function () {
-        $.getJSON(dashboardUrl, {action: 'raw'})
-            .done(function (json) {
+    $(function () {
+        $.ajax({
+            url: dashboardUrl,
+            data: {action: 'raw'},
+            success: function (json) {
                 dashboardViewModel.update(json);
-                if (!dashboardViewModel.bindingApplied()) {
-                    // We have to do this on the async ajax thread otherwise there
-                    // still might be a flicker on the page.
-                    $('#messaging_dashboard').koApplyBindings(dashboardViewModel);
-                    dashboardViewModel.bindingApplied(true);
-                }
+                $('#messaging_dashboard').koApplyBindings(dashboardViewModel);
+                dashboardViewModel.bindingApplied(true);
```

Comment:
I know you didn't add this, but we have `.ko-template` which is what we use elsewhere for this.

Reply:
This flag controls a loading message and spinner here, which is a slightly nicer UI that isn't easily supported by `.ko-template`, and I didn't have the heart to remove it.

Reply:
ah, lovely